service/telemetry/config.go

```diff
@@ -102,4 +102,7 @@ type MetricsConfig struct {
 
 	// Address is the [address]:port that metrics exposition should be bound to.
 	Address string `mapstructure:"address"`
+
+	// Namespace indicates the telemetry namespace the service will set. If nil, a default value is assigned.
```

Comment:
The description is not very clear. As an otel collector user, I don't understand it without looking into the code. Please mention that it changes namespace of exposed OTel collector's metrics and that default namespace is `otelcol`. It maybe good to also show an example of how the metrics will look like if the namespace is changed for a generic metric like `otelcol_process_uptime`

Reply:
that is fair. Will add it to be more explicit.

Reply:
added in the new version